toml/toml.go

```diff
@@ -112,8 +112,9 @@ func (m *FileMode) UnmarshalText(text []byte) error {
 func (m FileMode) MarshalText() (text []byte, err error) {
 	if m != 0 {
 		return []byte(fmt.Sprintf("%04o", m)), nil
+	} else {
+		return []byte(""), nil
 	}
-	return nil, nil
```

Comment:
Did you try running `go test ./...` without the changes to toml.go? I had to make the change because one of the package upgrades somehow broke a config test here, but maybe 2.x doesn't need it. Or maybe this is the right thing to do. I'm not sure.

Reply:
I did not experiment with the `toml.go`.  Your change merged without complaint, and all the tests passed, so I left it as you had modified.

Reply:
It seems to break lint, I think.

Reply:
Or something is breaking lint.